src/features/exports.test.ts

```diff
@@ -123,57 +123,91 @@ describe.concurrent('generateExports', () => {
     `)
   })
 
-  test('dts', async ({ expect }) => {
-    const results = generateExports(
+  test('dts with single entry', async ({ expect }) => {
+    const results = await generateExports(
       FAKE_PACKAGE_JSON,
       cwd,
       {
         es: [genChunk('foo.js'), genChunk('foo.d.ts')],
-        cjs: [genChunk('foo.cjs'), genChunk('foo.d.cts')],
       },
-      {},
+      { types: true },
     )
-    await expect(results).resolves.toMatchInlineSnapshot(`
-      {
+    expect(JSON.stringify(results, undefined, 2)).toMatchInlineSnapshot(`
+      "{
+        "main": "./foo.js",
+        "module": "./foo.js",
+        "types": "./foo.d.ts",
         "exports": {
-          ".": {
-            "import": "./foo.js",
-            "require": "./foo.cjs",
-          },
-          "./package.json": "./package.json",
-        },
+          ".": "./foo.js",
```

Comment:
I just added `dts with single entry` test, and `types` field should be added here.

Reply:
Wouldn't it make sense for the "types" setting to affect the [root-level types](https://github.com/rolldown/tsdown/blob/main/src/features/exports.ts#L182) too?

Reply:
In fact, setting `exports.types` itself doesn't make much difference, since TypeScript will look for `.d.ts` files based on the `.js` entry. However, explicitly specifying it may improve resolution speed. Regardless, since we have this option, we should set it for the main entry as well to maintain consistency.

Reply:
Thanks for your explanation.
If there's anything else I need to change, please feel free to let me know.